grpc.go

```diff
@@ -1193,6 +1216,12 @@ func (a *agentGRPC) CreateSandbox(ctx context.Context, req *pb.CreateSandboxRequ
 	a.sandbox.running = true
 	a.sandbox.sandboxPidNs = req.SandboxPidns
 	a.sandbox.storages = make(map[string]*sandboxStorage)
+	a.sandbox.guestHooks = &specs.Hooks{}
+	a.sandbox.guestHooksPresent = false
+
+	if req.GuestHookPath != "" {
+		a.sandbox.scanGuestHooks(req.GuestHookPath)
```

Comment:
The error is still silently ignored. I wonder how a user can find it out later on if `scanGuestHooks` fails here. I think we should at least report it back in `CreateSandboxResponse` and runtime can decide if it wants to bail out or continue using the hooks-missing image.

Reply:
Through the logs:
https://github.com/kata-containers/agent/pull/393/files/980023ec62b3144ef9f333209290c6215e295538#diff-2fe34e1b6f7f9aece0af74b1635502eeR259
I thought we agreed that it was fine to just log it for now. But I'm happy to change it if we have consensus.

Reply:
Yeah, I am leaning towards just logging this for now.

Reply:
Fair enough!